nginx/configuration_builder/templates/nginx_location_template.conf.j2

```diff
@@ -1,4 +1,4 @@
-    location ~ {{ route }}(.*)$ {
```

Comment:
If you remove this regex matching, would that impact current usage to match some paths that cannot be found?